base/db/DESCRIPTION

```diff
@@ -71,7 +72,8 @@ Suggests:
     rmarkdown (>= 2.19),
     testthat (>= 2.0.0),
     tidyverse,
-    withr
+    withr,
+    openssl
```

Comment:
Package added as a dependency, but I can't find any places in your PR where it's actually used. Can this be removed?